HELM_IMPROVEMENTS.md

```
@@ -0,0 +1,264 @@
+## 🔴 Critical Issue Fixed
+
+### Problem
+The original `persistentVolume.yaml` template contained an invalid Kubernetes schema:
+```yaml
+csi:
+  driver: efs.csi.aws.com
+  nfs:           # ❌ INVALID: CSI drivers don't have an nfs field
+  volumeHandle: fs-123456
+```
+
+**Error**: `field not declared in schema` when deploying Appsmith on AWS EKS with EFS enabled.
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)